src/docs/swagger.yaml

```
@@ -418,13 +535,19 @@ paths:
+                - profileImage
               properties:
                 profileImage:
                   type: string
                   format: binary
                   description: Profile image file (JPEG, PNG, or GIF, max 5MB)
-              required:
-                - profileImage
+              x-www-form-urlencoded:
+                type: object
+                properties:
+                  profileImage: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)